src/riscv-integration.adoc

```diff
@@ -506,10 +506,18 @@ interrupts cause the <<pcc>> to be set to the capability with
 its address incremented by four times the interrupt cause number.
 
 Capabilities written to <<mtvecc>> also include writing the MODE field in
-**mtvecc.address[1:0]**. As a result, a representability and sealing check is
-performed on the capability with the legalized (WARL) MODE field included in
-the address. The tag of the capability written to <<mtvecc>> is cleared if
-either check fails.
+**mtvecc.address[1:0]**, which is a WARL field, meaning that the capability address can
+change on writing. If the MODE field is non-zero after any WARL conversion,
+then the address used for the exception vector (which has **mtvecc.address[1:0]=2'b00**)
+will not match the value read by software from the CSR.
```

Comment:
I'm not sure about this - won't we always read the legalized value? Or do you mean won't match the value "written to the CSR"?

Reply:
New wording fixes this issue thanks.